call.go

```diff
@@ -52,14 +52,6 @@ func combine(o1 []CallOption, o2 []CallOption) []CallOption {
 	return ret
 }
 
-// Invoke sends the RPC request on the wire and returns after response is
-// received.  This is typically called by generated code.
-//
-// DEPRECATED: Use ClientConn.Invoke instead.
-func Invoke(ctx context.Context, method string, args, reply interface{}, cc *ClientConn, opts ...CallOption) error {
```

Comment:
We can never delete this function.  It is part of our public API and was not experimental / deprecated upon its introduction.